openmc/mgxs/mgxs.py

```diff
@@ -7,13 +7,14 @@
 import sys
 import copy
 import abc
+import itertools
 
 import numpy as np
 
 import openmc
 import openmc.checkvalue as cv
 from openmc.mgxs import EnergyGroups
-
+from openmc import Mesh
```

Comment:
Any reason to import `Mesh` on its own rather than just use `openmc.Mesh` throughout?